src/components/RecentSnippets.jsx

```diff
@@ -45,8 +45,6 @@ const RecentSnippets = ({ fetchRecentSnippets, pagination, snippets, recent }) =
     const marker = Number(pagination.get('next').marker)
 
     fetchRecentSnippets(marker)
-
-    scrollTop()
```

Comment:
hey @Shanmugapriya03 this line can't be removed so easily. 
The common way how it should work is: user clicks 'Older' (or 'Newer') and I want him to see snippets from very top of the new page, so I need `scrollTop()` function to be there.

Issue is a corner case here: we show 20 snippets on the page by default, BUT when there are less than 20 snippets left on the next page (f.e 2 or 3, small amount) and user clicks button to load them -> page tries to execute `scrollTop()` even if there is no need for this.

Reply:
Since we use live server (it will be very annoying to reach the end of the list) to reproduce this you might use Fiddler or something like it to mock `snippets` response with 21 snippet or so